.github/workflows/deploy.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       - uses: oven-sh/setup-bun@v1
       - name: Install dependencies
         run: bun install
+      - run: npx playwright install --with-deps chromium
```

Comment:
This was advised in the installation instructions: https://github.com/remcohaszing/rehype-mermaidjs